packages/communication-react/review/communication-react.api.md

```diff
@@ -2263,6 +2271,9 @@ export interface RemoteVideoStreamState {
     view?: VideoStreamRendererViewState;
 }
 
+// @public
+export type ScalingMode = 'Stretch' | 'Crop' | 'Fit';
```

Comment:
Is there a reason we're redefining this type instead of using @azure/communication-calling#ScalingMode. The values are identical, and presumably the only way we use this type is to forward it to the headless SDK.